web/libs/datamanager/src/components/Common/Table/TableHead/TableHead.jsx

```diff
@@ -218,7 +218,10 @@ export const TableHead = observer(
 
         useEffect(() => {
           ref.current?.addEventListener("mousedown", (event) => {
-            if (event.target.className.includes("handle")) event.preventDefault();
+            const className = event.target.className;
+            if (typeof className === "string" && className.includes("handle")) {
```

Comment:
```suggestion
            const className = event.target.className;

            if (typeof className === "string" && className.includes("handle")) {
```
linter eyes...

Reply:
We are in the after biome era, It is a rule that we should enable?

Reply:
we had this rule before: https://eslint.org/docs/latest/rules/newline-after-var
for some reason can't see it in biome rules